util/format/format_test.go

```diff
@@ -47,6 +48,7 @@ z
 	expect = "abc3%e x y z\n "
 	checkFormat(t, f, buf, str, expect)
 
-	str2 := OutputFormat(`\'\000abc\n\rdef`)
```

Comment:
the [back quote](https://go.dev/ref/spec#:~:text=character%20sequences%20between-,back%20quotes,-%2C%20as%20in%20%60foo) is a better way to write the case, no need to change this case